packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1071,6 +1037,14 @@ export const prepareVaultManagerKit = (
             /** @type {import('@agoric/zoe/src/contractSupport/atomicTransfer.js').TransferPart[]} */
             const transfers = [];
             let liquidated = 0;
+            /** @type {MapStore<string, Vault>} */
+            const vaultsToReinstate = makeScalarMapStore();
```

Comment:
between here and when these entries are read, there can be failure that exits the function. That in-between should be in a try/catch so execution continues to the reinstating and publishing.